package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+# Water Bridge Analysis
+r"""Water Bridge analysis --- :mod:`MDAnalysis.analysis.hbonds.WaterBridgeAnalysis`
+===============================================================================
+
+:Author: Zhiyi Wu
+:Year: 2017
+:Copyright: GNU Public License v3
+
+
+Given a :class:`~MDAnalysis.core.universe.Universe` (simulation
+trajectory with 1 or more frames) measure all water bridges for each
+frame between selections 1 and 2.
+Water bridge is defined as a bridging water which simutaneously form
+two hydrogen bonds with atoms from both selection 1 and selection 2.
+
+A water bridge can form between two hydorgen bond acceptors.
+
+e.g. -CO\ :sub:`2`\ :sup:`-`:···H−O−H···:\ :sup:`-`\ O\ :sub:`2`\ C-
+
+A water bridge can also form between two hydorgen bond donors.
+
+e.g. -NH···:O:···HN- (where O is the oxygen of a bridging water)
+
+A hydorgen bond acceptor and another hydorgen bond donor can be bridged by a water.
+
+e.g. -CO\ :sub:`2`\ :sup:`-`:···H−O:···HN- (where H−O is part of **H−O**\ −H)
+
+The :class:`WaterBridgeAnalysis` class is modeled after the  \ :class:`~MDAnalysis.analysis.hbonds.hbond_analysis.HydrogenBondAnalysis`.
+
+Options:
```

Comment:
Perhaps be clearer here and say instead of Options "The following keyword arguments are important to control the behavior of the water bridge analysis":

Reply:
@orbeckst Thank you for the comment. I sort of write this section based on the hbond_analysis. I'm a bit doubtful about the usefulness of this section as the kwargs are explained in full in the later sections.